### PR TITLE
fix(contrib/rackspace): add double quotes around POSIX vars

### PR DESCRIPTION
**Pull request by kzap**
put double quotes around paths incase running from osx or paths with spaces

### CODE REVIEW

**Comment by deis-admin** [comment]
Thanks for the contribution! Please ensure your commits follow our [style guide](http://docs.deis.io/en/latest/contributing/standards/#commit-style-guide). This code will be tested once a Deis maintainer reviews it.

File: contrib/rackspace/provision-rackspace-cluster.sh [hunk 1]
```diff
@@ -5,10 +5,10 @@
 
 set -e
 
-THIS_DIR=$(cd $(dirname $0); pwd) # absolute path
-CONTRIB_DIR=$(dirname $THIS_DIR)
+THIS_DIR="$(cd "$(dirname $0)"; pwd)" # absolute path
```
Comment (mboersma) [inline contrib/rackspace/provision-rackspace-cluster.sh]: Two sets of quotes should be enough here:
```bash
THIS_DIR=$(cd "$(dirname $0)"; pwd) # absolute path
CONTRIB_DIR=$(dirname "$THIS_DIR")
```
Then quote `"$CONTRIB_DIR" as you did below. I tested this on a Mac in a folder with spaces in the name.

**Comment by mboersma** [comment]
There are similar statements with `THIS_DIR` and `CONTRIB_DIR` in other `contrib/` scripts that should also be quoted, and as I mentioned above, I think the PR adds more quotes than necessary.

@kzap if you are willing to make similar changes wherever `THIS_DIR` and `CONTRIB_DIR` shows up, that would be very helpful for anyone who uses Deis code in a folder containing spaces. Or if that's more than you bargained for, just let us know and we can finish that up. Thanks!

**Comment by kzap** [comment]
@mboersma ok updated the other scripts but not sure if they work as i didnt test on those providers

**Comment by mboersma** [comment]
Refs #4044.

**Comment by carmstrong** [comment]
@kzap Are you able to rebase this against master? We've had a lot of changes lately. Thanks!

**Comment by bacongobbler** [comment]
@carmstrong I think this is superseded by #4044 which will catch this in the future :)

**Comment by mboersma** [comment]
@kzap my apologies; we really appreciate all the work you did here, but I'm going to close this in favor of #4044 which takes a more comprehensive approach to quoting shell variables. If you have time to review #4044 and double-check that it covers the same bases, we would value any feedback you have. Thanks again!

**Comment by kzap** [comment]
not sure on shellcheck, but if it will catch the errors then ok :)

**Pull request closed without merging**